scm-webapp/src/main/java/sonia/scm/api/v2/JsonMarshallingResponseFilter.java

```diff
@@ -59,11 +59,17 @@ public JsonMarshallingResponseFilter(ObjectMapper objectMapper, Set<JsonEnricher
   public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) {
     if (hasVndEntity(responseContext)) {
       JsonNode node = getJsonEntity(responseContext);
-      callEnrichers(requestContext, responseContext, node);
+      if (!isErrorNode(node)) {
```

Comment:
Isn't a check for the content type safer?

Reply:
You are right, this is much more simple